src/index.js

```diff
@@ -153,12 +170,162 @@ const IN_CALL_BUTTONS_ARRAY = [
             }
         }
 
+        const renderAddNumber = () => {
+            const modalElement = document.createElement("div")
+            modalElement.id = "modal"
+            modalElement.style.visibility = "visible"
+
+            const modalControlWrapperElement = document.createElement("div")
+            modalControlWrapperElement.id = "modal-control-wrapper"
+
+            const cancelButtonElement = document.createElement("button")
+            cancelButtonElement.innerHTML = "Cancel"
+            cancelButtonElement.className = "modal-controls"
+            cancelButtonElement.onclick = () => {
+                modalElement.style.visibility = "hidden"
+                firstNameElement.value = ""
+                lastNameElement.value = ""
+                emailElement.value = ""
+            }
+
+            const newContactTitleElement = document.createElement("p")
+            newContactTitleElement.innerHTML = "New Contact"
+            newContactTitleElement.className = "new-contact"
+
+            const doneButttonElement = document.createElement("button")
+            doneButttonElement.innerHTML = "Done"
+            doneButttonElement.className = "modal-controls"
+            doneButttonElement.onclick = () => {
+                modalElement.style.visibility = "hidden"
+                const newContact = {}
+                newContact.firstName = firstNameElement.value
+                newContact.lastName = lastNameElement.value
+                newContact.phoneNumber = phoneNumberElement.value
+                newContact.eMail = emailElement.value
+
+                CONTACTS.push(newContact)
```

Comment:
const newContact = {
  firstName:firstNameElement.value,
  lastName: lastNameElement.value,
  phoneNumber: phoneNumberElement.value,
   email: emailElement.value
}

Would be much cleaner